### DISCUSSION TITLE
How is the client used?

### DISCUSSION THREAD

**Discussion by neofof234** [projectdiscovery/interactsh#192] [General]
Install the client in kali as directed by GOi, find the directory where it is installed and run it. The token or authorization is uploaded and provided. I use the token on a website https://www.site.com.XXXXX.interact.sh and in the client window I display the initial requests. But after that what? install in ZAP as OAST using your server, do automatic and manual scan and see no results.
How is it used?

**Comment by MetzinAround** [comment]
Closing issue that is over a year old. Feel free to reopen if you are still in need of help!
